definitions/variable/tag_emission_kind.yaml

```
@@ -0,0 +1,7 @@
+- Emission Kind:
```

Species is a more common name for different types of emissions or greenhouse gases...
```suggestion
- Emission Species:
```